geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -78,6 +83,20 @@ def get_http_cached(http_options, url, headers):
     return response
 
 
+@CACHE_REGION_OBJ.cache_on_arguments()
+def _build_web_map_service(no_cache, ogc_server_id):
+    # The content is named no_cache to don't be concidered in the cache key
```

Comment:
not to be considered

note that we should consider adding parameters to the decorator to ignore some parameters, it would be more understandable, example:
```
@CACHE_REGION_OBJ.cache_on_arguments(ignore_args=['xml'])
def _build_web_map_service(xml, ogc_server_id):
```

Reply:
and more flexible when we would have two custom parameters to ignore.

Reply:
But it's not our decorator, it's used by keygen_function who hasn't a know leg of the method, if you have an idea it's welcome :-)